extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -162,9 +186,26 @@ class AmpIframe extends AMP.BaseElement {
         this.updateHeight_(newHeight);
       }
     });
+    if (this.isClickToPlay_) {
+      listen(iframe, 'embed-clickready', () => {
+        this.activateIframe_();
+      });
+    }
     return loadPromise(iframe);
   }
 
+/**
+ * Makes the iframe visible.
+ * @private
+ */
+  activateIframe_() {
+    if (this.placeholder_) {
+      this.iframe_.style.zIndex = 'auto';
+      removeElement(this.placeholder_);
```

Comment:
Any reason why we would want to retain the element on the DOM? i dont see any use for it after we show the iframe.